tests/performance/checkboxTreeView/src/main/java/checkboxTreeView/CheckBoxTreeEditor.java

```diff
@@ -0,0 +1,89 @@
+package checkboxTreeView;
```

Comment:
This needs a copyright header. Also, the convention for package names is to use all lower case.